generators/typescript/utils/abstract-generator-cli/src/loadIntermediateRepresentation.ts

```diff
@@ -1,10 +1,9 @@
 import { IntermediateRepresentation } from "@fern-fern/ir-sdk/api";
 import * as IrSerialization from "@fern-fern/ir-sdk/serialization";
-import { readFile } from "fs/promises";
+import { AbsoluteFilePath, streamObjectFromFile } from "@fern-api/fs-utils";
 
 export async function loadIntermediateRepresentation(pathToFile: string): Promise<IntermediateRepresentation> {
-    const irString = (await readFile(pathToFile)).toString();
-    const irJson = JSON.parse(irString);
+    const irJson = await streamObjectFromFile(AbsoluteFilePath.of(pathToFile));
```

Comment:
I've not been able to test this locally. I did try following the instructions in `CONTRIBUTING.md`, however the process output a lot of errors and the docker container does not seem to do anything but open a shell. I'm obviously doing something wrong, but it's not clear to me what.

Reply:
I believe this issue is common to all generators, so other `loadIntermediateRepresentation` functions should probably be updated the same way.